m2cgen/interpreters/interpreter.py

```diff
@@ -9,12 +9,13 @@ class BaseInterpreter:
     which takes instance of AST expression and recursively applies method
     _do_interpret() to it.
     """
+    def __init__(self):
+        self._cached_expr_results = {}
 
     def interpret(self, expr):
+        self._reset_reused_expr_cache()
         return self._do_interpret(expr)
 
-    # Private methods implementing Visitor pattern
-
```

Comment:
Was this comment intentionally deleted? I think it adds some clarity about what exactly we are doing here.

Reply:
nope, accident